ot3_state_manager/ot3_state_manager/messages.py

```diff
@@ -5,102 +5,156 @@
+        )
+
+    def __hash__(self) -> int:
+        """Return hash for instance."""
+        return hash(str(self))
+
+    @staticmethod
+    def build(message_content: bytes) -> MoveMessage:
+        """Convert message_content into a MoveMessage object."""
+        hw_id, direction_val = struct.unpack(">HB", message_content)
+        # mypy thinks that I am instantiating a MoveMessageHardware object here.
+        # I am not, I am performing a lookup as defined in the enum docs.
+        # https://docs.python.org/3/library/enum.html#programmatic-access-to-enumeration-members-and-their-attributes
+        axis = MoveMessageHardware(hw_id).axis  # type: ignore[call-arg]
+        direction = Direction.from_int(direction_val)
+        return MoveMessage(axis=axis, direction=direction)
+
+    def to_bytes(self) -> bytes:
+        """Convert MoveMessage object into a sequence of hexadecimal bytes."""
+        hw_id = MoveMessageHardware.from_axis(self.axis).hw_id
+        return struct.pack(">BHB", MessageID.MOVE.message_id, hw_id, self.direction)
 
 
 @dataclass
 class SyncPinMessage(Message):
     """Message for setting sync pin high or low."""
 
-    state: Literal["HIGH", "LOW"]
-
-    @classmethod
-    def to_message(cls, string: str) -> Optional[SyncPinMessage]:
-        """Parse string into SyncPinMessage or None."""
-        if string.startswith("SYNC_PIN"):
-            split_string = string.split(" ")
-            if len(split_string) == 2:
-                sync_pin_state = split_string[1]
-                if sync_pin_state == "HIGH":
-                    return cls(state="HIGH")
-                elif sync_pin_state == "LOW":
-                    return cls(state="LOW")
-
-        return None
-
-
-@dataclass
-class MoveMessage(Message):
-    """Message to be sent between hardware simulators and OT3StateManager"""
-
-    axis: OT3Axis
-    direction: Direction
+    state: SyncPinState
 
-    @classmethod
-    def to_message(cls, string: str) -> Optional[MoveMessage]:
-        """Parse string to Message or None."""
-        split_string = string.split(" ")
+    def __eq__(self, other: Any) -> bool:
+        """Confirm that the 2 objects are equal"""
+        return isinstance(other, SyncPinMessage) and other.state == self.state
 
-        if len(split_string) == 2:
-            try:
-                axis = OT3Axis[split_string[1]]
-            except KeyError:
-                axis = None
+    def __hash__(self) -> int:
+        """Return hash for instance."""
+        return hash(str(self))
 
-            try:
-                direction = Direction.from_symbol(split_string[0])
-            except ValueError:
-                direction = None
+    @staticmethod
+    def build(message_content: bytes) -> SyncPinMessage:
+        """Convert message_content into a SyncPinMessage object."""
+        # Have to use from_bytes method because struct package won't accept a 3 byte val
```

Comment:
that seems a bit annoyingly inconsistent - perhaps we should just make it four bytes

Reply:
If I make message_content 4 bytes then the hw_id portion of the MoveMessage becomes 3 bytes which leads me to the same issue. 
Instead of making it 4 bytes, I am going to prepend a `\x00` to the `message_content` of the `build` method of `SyncPinMessage`. This will make it 4 bytes, making it parseable by the struct module, without changing the integer value of it

Reply:
e33ed939761988879d0989afccd96b647c613005